common/djangoapps/student/views.py

```diff
@@ -893,6 +884,55 @@ def dashboard(request):
     return response
 
 
+def course_refund_status(request, course_id):
+    """
+    Get Refundable status for a course.
+
+    Arguments:
+        request: The request object.
+        course_id (str): The unique identifier for the course.
+
+    Returns:
+        Json response.
+
+    """
+    # get course key
+    from django.http import JsonResponse
+
+    if course_id:
+        try:
+            course_key = CourseKey.from_string(course_id)
+            course_enrollment = CourseEnrollment.get_enrollment(request.user, course_key)
+
+        except InvalidKeyError:
+            logging.exception("An error as occured while getting refund status for a course")
+
+            return JsonResponse({
+                'course_refundable_status': '',
+                'success': False
```

Comment:
Returning a success field is redundant with the status code. Remove this field and rely on the status code.